sparkle/src/_index.ts

```diff
@@ -25,6 +25,9 @@ export { ContextItem };
 import { AssistantPreview } from "./components/AssistantPreview";
 export { AssistantPreview };
 
+import { AssistantPreview2 } from "./components/AssistantPreview2";
```

Comment:
That's the plan. I just think it's easier to transition no?

Reply:
Ok! Don't think it's necessary; if the new component is usable in place of the old one, then I suggest directly removing this one, then following up shortly PR updating front using the new version of sparkle (available to help if you think it's a pain)

But maybe I missed a complexity? available to discuss if you believe it will really help things

Reply:
> if the new component is usable in place of the old one, then I suggest directly removing this one, then following up shortly PR updating front using the new version of sparkle (available to help if you think it's a pain)

It's breaking though. It requires implementing new lists to work

Reply:
OK GFM